### PR TITLE
Valentine's day thing

### PR DESCRIPTION
**Pull request by ghost**
:cl: McBawbaggings
tweak: Recent intel suggests that enemies of Central Command have undertaken training to not be heart broken on Valentine's day.
/:cl:

https://github.com/tgstation/tgstation/pull/35678
Was denied upstream but it certainly conflicts with our own ruleset anyway.
Since this is a world accessible proc, it cannot be modularised.

### CODE REVIEW

**Comment by KayeArray** [comment]
kill /tg/ the movie

File: code/modules/events/holiday/vday.dm [hunk 1]
```diff
@@ -49,17 +49,23 @@
 			L.mind.objectives += normiesgetout
 			L.mind.add_antag_datum(/datum/antagonist/auto_custom)
 
+// hippie start - Antagonists shouldn't be getting messages that restrict their play on our ruleset.
```
Comment (tortellinitony) [inline code/modules/events/holiday/vday.dm]: uh no just mirror this proc

**Review by tortellinitony — DISMISSED** [review]

File: code/modules/events/holiday/vday.dm (hunk 1 above)
Comment (ghost) [reply code/modules/events/holiday/vday.dm]: can't!
duplicate definition error

**Comment by ghost** [comment]
@tortellinitony unless you want me to disable the /tg/ file and recreate the valentine event in the hippie folder and add that to hippiestation.dmb

**Comment by tortellinitony** [comment]
That might be worth it so then we can make our own candy hearts and stuff. The file is not very big.

**Comment by ghost** [comment]
alright will do

File: hippiestation/code/modules/events/holiday/vday.dm [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+			forge_valentines_objective(date, L)
+
+			if(valentines.len && prob(4))
+				var/mob/living/notgoodenough = pick_n_take(valentines)
+				forge_valentines_objective(notgoodenough, date)
+
+
+		else
+			to_chat(L, "<span class='warning'><B>You didn't get a date! They're all having fun without you! you'll show them though...</B></span>")
```
Comment (JohnGinnane) [inline hippiestation/code/modules/events/holiday/vday.dm]: Trivial complaint but can you change "you'll show them" to "**Y**ou'll show them"

**Review by JohnGinnane — DISMISSED** [review]
Looks p good

File: hippiestation/code/modules/events/holiday/vday.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/modules/events/holiday/vday.dm]: Funnily enough, I didn't change the maytr shit, that's how it be from /teegee/ but I'll fix it.

Comment (JohnGinnane) [reply hippiestation/code/modules/events/holiday/vday.dm]: That's what I figured..

**Comment by Pyko1** [comment]
+1

**Comment by Carbonhell** [comment]
+1

**Pull request merged**